corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -78,8 +78,8 @@ hqDefine("users/js/mobile_workers", function () {
             return self.isLoaded() && !self.hasError() && !self.showNoUsers() && !self.showProjectHasNoUsers();
         });
 
-        self.toggleInactiveOnly = function (inactiveOnly) {
-            self.inactiveOnly(inactiveOnly);
+        self.setDeactivatedOnly = function (deactivatedOnly) {
+            self.deactivatedOnly(deactivatedOnly);
```

Comment:
I still think it's preferable to just set this property directly in the view and set up a listener, that way any future travelers to this code don't have to figure out that they need to use this setter instead. But up to you if you want to keep it this way.

Reply:
e80ec1b

Also did something similar for `is_active`, replacing the `modifyStatus` with a subscription, because initially I read this comment over email and thought you were talking about `is_active` not `deactivatedOnly`.